README.md

```diff
@@ -41,14 +40,35 @@ USAGE
 ...
 ```
 <!-- usagestop -->
-# Commands
+
+## Configuration
+
+`wrhs` configuration can be passed in via CLI flags (more information on this
+below), or by creating a configuration file at `~/.wrhs`. _Note: by default
+the CLI expects the configuration file to be available under your home folder._
+
+### Example `.wrhs` file:
+```
+{
+  "hosts": {
+    "wrhs": "warehouse.ai",
+    "status": "warehouse-status.ai"
+  },
+  "auth": {
+    "user": "username",
+    "pass": "password"
+  }
+}
+```
+
+## Commands
 <!-- commands -->
 * [`wrhs get:build PACKAGE ENV [LOCALE]`](#wrhs-getbuild-package-env-locale)
 * [`wrhs get:head PACKAGE ENV`](#wrhs-gethead-package-env)
 * [`wrhs get:status PACKAGE ENV`](#wrhs-getstatus-package-env)
 * [`wrhs help [COMMAND]`](#wrhs-help-command)
 
-## `wrhs get:build PACKAGE ENV [LOCALE]`
+### `wrhs get:build PACKAGE ENV [LOCALE]`
```

Comment:
unfortunately I think these docs are all generated by oclif and this'll just get stomped next time its run.  I'm not sure if there's a way to customize it. :/

Reply:
Good callout, I don't mind if it changes the header size. But I would be dissapointed if some of the other sections are gone. I'll double check. Perhaps it is as simple as adding an opt-out comment or removing the comment sections.